apps/artist/client/views/header.coffee

```diff
@@ -2,7 +2,7 @@ _ = require 'underscore'
 Backbone = require 'backbone'
```

Comment:
I simplified this, basically we need to do something bit different in the follow button (when we're in this test).